backend/src/modules/tournament-details/core/domain/TournamentDetails.ts

```diff
@@ -0,0 +1,19 @@
+import { TournamentName } from './TournamentName';
+
+export class TournamentDetails {
+  readonly tournamentId: string;
+  readonly tournamentName: TournamentName;
+
+  constructor(props: { tournamentId: string; tournamentName: TournamentName }) {
+    this.tournamentId = props.tournamentId;
+    this.tournamentName = props.tournamentName;
+  }
+}
+
+export function addTournamentDetails(command: { tournamentId: string; tournamentName: string }): TournamentDetails {
+  if (command.tournamentName.length < 3) {
```

Comment:
Po co to tutaj? Wystarczy to dać w TournamentName :) To nasza logika domenowa w takim razie, ze nie istnieje TournamentName, ktore ma mniej niz 3 znaki.

Reply:
aaa tak myślałem gdzie to dać i padło tutaj, żeby ta domena taka smutna i pusta nie była xD poprawię!